dev/com.ibm.ws.javamail.1.6/src/com/ibm/ws/javamail/internal/injection/MailSessionResourceFactoryBuilder.java

```diff
@@ -94,34 +81,10 @@ public ResourceFactory createResourceFactory(Map<String, Object> props) throws E
         if (!removeExistingConfigurations(filter.toString()))
             throw new IllegalArgumentException(mailSessionID); // internal error, shouldn't ever have been permitted in server.xml
 
-        mailSessionSvcProps.put(MailSessionService.JNDI_NAME, jndiName);
```

Comment:
Need to also leave in the puts for the four properties you removed above.  jndiName here...

Reply:
updated